CONTRIBUTING.md

```diff
@@ -106,11 +106,11 @@ I personally do not like something that requires so many configurations before y
 
 ## Tools
 
-- Node.js >= 14
-- NPM >= 8.5
-- Git
-- IDE that supports ESLint and EditorConfig (I am using IntelliJ IDEA)
-- A SQLite GUI tool (SQLite Expert Personal is suggested)
+- [`Node.js`](https://nodejs.org/de) >= 14
```

Comment:
should be `https://nodejs.org/` (no lang at the end).

Reply:
```suggestion
- [`Node.js`](https://nodejs.org/) >= 14
```